src/Informer/InfoAtLinkExtractFlags.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("compiler")
```

Comment:
some comments as to why you use argparse and these arguments would be helpful, I think